testsuite/azure-arc-conformance.properties

```diff
@@ -37,7 +37,8 @@ azure-arc-platform.CUSTOM_LOCATION_OID=8e103bf2-1f6a-43d9-9473-52a2461574b4
 #azure-arc-agent-cleanup.PROXY_CERT_NAMESPACE="namespace of sonobuoy secret"
 
 # azure-arc-ds-connect-platform plugin configurations
-# Set below to true if you want to run the tests for data services in direct mode.
+# To perform data services with direc connect mode please enable azure-arc-platform.enable=true
+# Set below to true if you want to run the tests for data services in direct connect mode.
```

Comment:
Line 40 and 41 are saying the same thing.

Reply:
Line 40 is prerequisite for data services with connect mode plugin.
Line 41 is just informing to enable azure-arc-ds-connect-platform plugin for data services with connect mode plugin.